packages/form-state-manager/src/utils/manager-api.ts

```diff
@@ -7,6 +7,8 @@ import FieldState from '../types/field-state';
 
 const isLast = (fieldListeners: AnyObject, name: string) => fieldListeners?.[name]?.count === 1;
 
+const addIfUnique = (array: Array<any>, item: any) => !array.includes(item) && array.push(item);
```

Comment:
I feel like this should be `Array<string>`. We should try to use `any` as little as possible.